src/adapters/opstack.ts

```diff
@@ -1,5 +1,6 @@
 import { Config, L2Transaction, L1ForceTransaction } from '../types';
-import { encodeFunctionData } from 'viem';
+import { encodeFunctionData, Hash, TransactionReceipt } from 'viem';
+import { getL2HashFromL1DepositInfo, getTransactionDepositedEvents } from 'op-viem';
```

Comment:
Q: can't viem's `extractTransactionDepositedLogs` and `getL2TransactionHashes` achieve the same purpose?
- https://viem.sh/op-stack/utilities/extractTransactionDepositedLogs
- https://viem.sh/op-stack/utilities/getL2TransactionHashes

Reply:
They seem to be deprecated in the latest version so I can't find them now 😢, so I have to switch to the new ones 
([search result](https://github.com/search?q=repo%3Abase-org%2Fop-viem%20extractTransactionDepositedLogs&type=code))

Reply:
updated with `viem/opstack`!